CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On Wayland, fix polling during consecutive `EventLoop::run_return` invocations.
 - On Windows, fix race issue creating fullscreen windows with `WindowBuilder::with_fullscreen`
 - On Android, `virtual_keycode` for `KeyboardInput` events is now filled in where a suitable match is found.
+- **Breaking:** Replaced `Window::with_app_id` and `Window::with_class` with `Window::with_name`.
```

Comment:
Use `WindowBuilderExtUnix` here to indicate that this change only affects those platforms.